04-spatial-operations.Rmd

```diff
@@ -176,13 +170,133 @@ row.names(africa[africa$subregion == "Northern Africa", ])
 row.names(filter(africa, subregion == "Northern Africa"))
 ```
 
```

Comment:
I have placed the topological relations directly after the subsetting operations because subsetting is actually a subgroup of the topological operations. Nevertheless, I would suggest to just show the intersection example, and maybe one further example. I think referencing the figure showing the differences between the topological operations should suffice.